src/renderer/Voice.tsx

```diff
@@ -154,15 +175,42 @@ export default function Voice() {
 		socket.on('connect', () => {
 			setConnected(true);
 		});
+
 		socket.on('disconnect', () => {
 			setConnected(false);
 		});
 
+		let iceConfig: RTCConfiguration = DEFAULT_ICE_CONFIG;
+		socket.on('peerConfig', (peerConfig: PeerConfig) => {
+			if (validatePeerConfig(peerConfig)) {
```

Comment:
Small style nit, I think it's better to check if not valid, error and break out (return) up front and then have the normal flow outside of a conditional (this is consistent with the rest of the project, I think)

Reply:
Good call, I'll update it.